### DISCUSSION TITLE
Support otpauth-migration export QR for Google Authenticator App

### DISCUSSION THREAD

**Discussion by Sneezry** [Authenticator-Extension/Authenticator#622] [General]
<You can also suggest feedback by Tweeting with #AuthenticatorFeedback>

**Describe the feature you want:**
We have already supported to import data from Google Authenticator App. We should also support export data from the extension in Google's private otpauth-migration format to allow Google Authenticator App to import our backups.

**Why do you want this feature in Authenticator?**
Google Authenticator is the most famous OTP App for moble phones and tablets. To support otpauth-migration format in two way (export and import) is useful to help users to handle their data better.

**Comment by alex-pancho** [comment]
Maybe it's help for you: https://github.com/google/google-authenticator-android/issues/118

**Comment by ira-at-work** [comment]
> We have already supported to import data from Google Authenticator App

where is that documented? I tried to scan an export from google auth but the extension ignored it.
